train.py

```diff
@@ -8,11 +8,20 @@
 import subprocess
 import tempfile
 
+from build_sym import _char_processor
 from typing import Set
 
 import pynini
 import pywrapfst
 
+def _type_reader(_type: str) -> pynini.SymbolTable or str:
+    """Allows for token_type from a SymbolTable text file
+    """
+    if _type not in ["byte", "utf8"]:
+        _token_type = pynini.SymbolTable.read_text(_type)
```

Comment:
just make this a return statement instead of assigning to a weirdly-named variable, and do the same on line 23.